mltsp/featurize_tools.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+import pandas as pd
+import xray
+import os
+import tarfile
+import tempfile
+import zipfile
+from mltsp import cfg
+from mltsp import custom_exceptions
+from mltsp import util
+from mltsp import obs_feature_tools as oft
+from mltsp import science_feature_tools as sft
+from mltsp import custom_feature_tools as cft
+
+
+def featurize_single_ts(t, m, e, custom_script_path, features_to_use,
```

Comment:
Yeah, I still feel a bit weird about `featurize` vs. `featurize_tools` as I don't really know how to describe what the difference between the two is. I think this issue will go away when we have separate logic for file system management, since splitting things out that way should eliminate the need for this distinction.